test/position/sync/chooseOne.test.js

```diff
@@ -3,14 +3,28 @@ const ChooseOne = require('../../../position/sync/chooseOne')
 const isPosition = require('../../../isPosition')
 
 test('Position - ChooseOne', function (t) {
-  var invalidPosition = ChooseOne({
+  var missingChoice = ChooseOne({
+    poll: '%t+PhrNxxXkw/jMo6mnwUWfFjJapoPWxzsQoe0Np+nYw=.sha256'
   })
-  t.false(isPosition(invalidPosition), 'missing a choice')
+  t.false(isPosition(missingChoice), 'missing a choice')
+
+  var missingPoll = ChooseOne({
+    choice: 0
+  })
+  t.false(isPosition(missingPoll), 'missing a poll')
+
+  var brokenPoll = ChooseOne({
+    poll: 'dog?',
+    choice: 0
+  })
+  t.false(isPosition(missingPoll), 'does not reference a poll')
 
   var validPosition = ChooseOne({
+    poll: '%t+PhrNxxXkw/jMo6mnwUWfFjJapoPWxzsQoe0Np+nYw=.sha256',
     choice: 0
   })
```

Comment:
I think a valid position needs to reference back to it's parent poll.
Not sure if this attribute should be `poll` or `root` (which might be more consistent and generic)

Reply:
I reckon root for now.